nym-vpn-core/Cargo.lock

```diff
@@ -528,7 +528,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "33415e24172c1b7d6066f6d999545375ab8e1d95421d6784bdfff9496f292387"
 dependencies = [
  "bitcoin_hashes",
- "rand 0.8.5",
+ "rand 0.7.3",
```

Comment:
Interesting that a bunch of crates were downgraded?

Reply:
I just ran `cargo update nym-authenticator-requests`, not sure why they would get downgraded. But I will remove those changes, no need to go downwards